LDMP/models/algorithms.py

```diff
@@ -131,7 +131,8 @@ def columnCount(self) -> int:
     def rowCount(self) -> int:
         if self.details is None:
```

Comment:
```suggestion
        return 0
```

Reply:
this method should just `return 0`, I feel the `if self.details is None:` is redundant

Reply:
I leaved dinstintion of the return just as trace in case want to go back adding subtree with more details.
I would leave the code in thsi way to explict rememeber to do something in case more datails are available . In this case no add any leaf.
If it's important I can simplify, but IMHO is useful for future maintainer of the code.

Reply:
I resolve conversation, feel free to open and I'll apply the changes if you disagree with my consideartions